Makefile

```diff
@@ -205,6 +205,29 @@ $(foreach p,$(SUBPROJECTS),$(eval $(call post-build-template,$(p))))
 .PHONY: post-build
 post-build: $(foreach p,$(SUBPROJECTS),post-build-$(p))
 
+###############################################################################
+### dev
+### Starts services in development mode with
+### `ln -s docker-compose.dev.yml docker-compose.override.yml; docker-compose up -d`
+###############################################################################
+define dev-template
+dev-$(1):
+	@if [ -e "$(1)/docker-compose.dev.yml" ]; then \
+	  echo "Starting $(1) in development mode" \
+	  && cd $(1) \
+		&& rm -rf docker-compose.override.yml \
+		&& ln -s docker-compose.dev.yml docker-compose.override.yml \
+		&& docker-compose up -d; \
+	else \
+	  echo "Starting $(1) from docker image" \
+	  && cd $(1) \
+		&& docker-compose up -d; \
```

Comment:
I think if a service is already running in non-dev mode, this won't have any effect. Might need a `docker-compose down` as well.

Reply:
ah, didn't test this one. thanks